data/db_demo.yaml

```diff
@@ -491,6 +499,18 @@ instrument:
     field_fov_attributes: 0.38
     # field_data:
     #   file: ../data/GROWTH-India_Fields.csv.bz2
+  - band: optical
+    name: WINTER
+    telescope_id: =WINTER-1m
+    type: imager
+    =id: WINTER
+    api_classname: GENERICAPI
+    api_classname_obsplan: MMAAPI
+    filters: ["desy", "2massj", "2massh"]
```

Comment:
What does "desy" mean here?

Reply:
desy is the dark energy survey y, we just need to choose the closest for now.

Reply:
Sure, that sounds reasonable!